mod/node-core/pkg/components/module/depinject.go

```diff
@@ -122,9 +67,43 @@ type DepInjectOutput struct {
 
 // ProvideModule is a function that provides the module to the application.
 func ProvideModule(in DepInjectInput) (DepInjectOutput, error) {
+	// TODO: this is hood as fuck.
+	if in.BeaconConfig.KZG.Implementation == "" {
```

Comment:
You should be able to remove this check now @ocnc2

Reply:
I'm too speedy LFG

Reply:
how r u so fast i literally removed it frame 1